geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportParams.scala

```diff
@@ -11,37 +11,39 @@ package org.locationtech.geomesa.tools.export
 import java.io.File
 
 import com.beust.jcommander.{Parameter, Parameters}
-import org.locationtech.geomesa.tools.{CatalogParam, OptionalCqlFilterParam, RequiredTypeNameParam}
+import org.locationtech.geomesa.tools.{CatalogParam, OptionalCqlFilterParam, RequiredTypeNameParam, TypeNameParam}
 
-trait BaseExportParams extends CatalogParam with RequiredTypeNameParam with OptionalCqlFilterParam {
+trait RootExportParams extends OptionalCqlFilterParam {
```

Comment:
Yeah, it's to get the type checking to work everywhere. There are a number of methods that are used by different commands that only require a subset of params. There might be a better way but it's not immediately obvious to me.